zulip/integrations/twitter/twitter-bot

```diff
@@ -43,7 +44,7 @@ that will process tweets every 5 minutes is:
 
 == Setting up Twitter authentications ==
 
-Run this on a personal or trusted machine, because your API key is
+Run this on a personal or trusted machine, because your APIx key is
```

Comment:
This is presumably a typo.

Reply:
Yes it is.
I will remove the x again.

Thanks